indexer/db/neon_tx_db.py

```diff
@@ -264,13 +309,19 @@ def from_tx(cls, neon_tx: NeonTxModel, neon_rcpt: NeonTxReceiptModel) -> Self:
             tx_idx=neon_rcpt.neon_tx_idx,
             neon_sig=neon_tx.neon_tx_hash.to_string(),
             tx_type=neon_tx.tx_type,
+            chain_id=neon_tx.tx_chain_id,
             from_addr=neon_tx.from_address.to_string(),
             nonce=hex(neon_tx.nonce),
             to_addr=neon_tx.to_address.to_string(None),
             contract=neon_tx.contract.to_string(None),
             value=hex(neon_tx.value),
             calldata=neon_tx.call_data.to_string(),
-            gas_price=hex(neon_tx.gas_price),
+            gas_price=hex(NeonTxMetaModel(neon_tx=neon_tx, neon_tx_rcpt=neon_rcpt).effective_gas_price),
```

Comment:
This method is called from one place, and in the source line there is `NeonTxMetaModel` object.
Maybe it is better to pass the base object, instead of constructing it from the params?

Reply:
I'm not sure I understand it. Yes, `from_tx` is only called from `set_block_list`, but there's no NeonTxMetaModel.

I created an additional `@classmethod` that allows to do the same logic but without constructing class object. The `@cached_property` is also in place to support access to `effective_gas_price` from `np_block_transaction_api`.

WDYT?